h/services/profile_group.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, session, route_url, open_group_finder):
         self.route_url = route_url
         self.open_group_finder = open_group_finder
 
+    def sort(self, groups):
+        """ sort a list of groups of a single type """
```

Comment:
I would just say "Sort a list of group", this method doesn't care whether they're a single type or not (although as it happens they are)